src/main/java/uk/gov/justice/hmpps/prison/service/MovementUpdateService.java

```diff
@@ -46,7 +45,7 @@ public MovementUpdateService(
     }
 
     @Transactional
-    @VerifyBookingAccess(overrideRoles = {"ROLE_SYSTEM_USER", "ROLE_UNILINK"})
+    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "UNILINK"})
```

Comment:
this doesn't make any difference, but removed the `ROLE_` prefix to conform to other usages